multipass/Linux.md

```diff
@@ -75,11 +75,11 @@ To check for new updates run: sudo apt update
 ubuntu@primary:~$
 ```
 
-Let's test it out! As we just learned, the previous step automatically mounted our $HOME directory in the instance. Let's use this to share data with our instance. More concretely, let's create a new folder in our $HOME directory called Multipass_Files:
+Test it out. As you've just learnt, the previous step automatically mounted your $HOME directory in the instance. Use this to share data with your instance. More concretely, create a new folder called Multipass_Files in your $HOME directory:
```

Comment:
I would personally keep "Let's test it out!" here, to keep the tone light and conversational :)